packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -100,9 +106,10 @@ const normalizeSettings = ({ settings, exportMode, copyMode, snapshotMode }) =>
       ? {
           copyRetention: copyMode ? setting.copyRetention : undefined,
           exportRetention: exportMode ? setting.exportRetention : undefined,
-          snapshotRetention: snapshotMode
-            ? setting.snapshotRetention
-            : undefined,
+          snapshotRetention:
+            snapshotMode && !offlineBackupEnabled
+              ? setting.snapshotRetention
+              : undefined,
```

Comment:
Can you explain this, please?

Reply:
We keep in the state the snapshot retention if it's edited by a user. 
If a user edit the snapshot retention then disable the snapshot mode and enable the offline backup mode , the `snapshotRetetion` should not be kept.